tests/admin/test-class-admin-asset-dev-server-location.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+class Test_Admin_Asset_Dev_Server_Location extends PHPUnit_Framework_TestCase {
```

Comment:
Missing documentation. Additionally, I see that this test class isn't `final` whereas the previous one is. Why is that?

Reply:
Because I'd created this class without using PHPStorm's method of adding a Class file.